lighthouse-core/gather/driver.js

```diff
@@ -477,11 +478,15 @@ class Driver {
     const waitForLoad = _options.waitForLoad || false;
     const disableJS = _options.disableJavaScript || false;
     const pauseAfterLoadMs = (_options.flags && _options.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
+    let maxWaitMs = MAX_WAIT_FOR_FULLY_LOADED;
+    if (_options.flags && _options.flags.timeout) {
+      maxWaitMs = 1000 * _options.flags.timeout;
```

Comment:
will do, anything magic I'm not aware of or just style? I know in this case it doesn't matter but I developed a habit of putting the type to coerce to first when dealing with potentially unknown types to avoid `{} + ''` vs `'' + {}` scenarios :)